### PR TITLE
Current month view

### PR DESCRIPTION
**Pull request by NeonLexie**
everything is working correctly now. Buttons might need to be restyled later

### CODE REVIEW

File: src/app/Components/calendar-view/calendar-view.component.ts [hunk 1]
```diff
@@ -37,14 +41,17 @@ export class CalendarViewComponent implements OnInit {
...
   getWeekdayForFirstOfMonth(month: number) {
-    const day = new Date(this.year + '-' + month + '-01').getDay();
+    let day;
+
+    if (month <= this.month.id) {
+      day = new Date(this.year + '-' + month + '-01').getDay();
+    } else {
+      day = new Date(this.year + '-' + month + '-01').getDay() + 1;
+    }
+
```
Comment (tan-cy) [inline src/app/Components/calendar-view/calendar-view.component.ts:54]: what is this part for?

File: src/app/Components/calendar-view/calendar-view.component.css [hunk 2]
```diff
@@ -9,6 +9,10 @@ header {
   min-height: 10vh;
   text-align: center;
 }
+div.arrows{
+margin-left: 2em;
+margin-bottom: 00.5em;
+}
```
Comment (tan-cy) [inline src/app/Components/calendar-view/calendar-view.component.css:15]: please keep formatting consistent

File: src/app/Components/calendar-view/calendar-view.component.html [hunk 3]
```diff
@@ -1,19 +1,32 @@
 <div>
-  <div *ngFor="let monthWithDay of monthsWithDays" class="calendar-wrapper">
+  <link
+    rel="stylesheet"
+    href="https://fonts.googleapis.com/icon?family=Material+Icons"
+  />
+  <!-- <button (click)="buttonClick()">Click for Month</button> -->
+  <!-- *ngFor="let monthWithDay of monthsWithDays" -->
```
Comment (tan-cy) [inline src/app/Components/calendar-view/calendar-view.component.html]: please remove unneeded code

File: src/app/Components/calendar-view/calendar-view.component.ts (hunk 1 above)
Comment (NeonLexie) [reply src/app/Components/calendar-view/calendar-view.component.ts:54]: this is the only way I could get the days to match up for any month after the current month. It was consistently off by one week day

File: src/app/Components/calendar-view/calendar-view.component.ts [hunk 4]
```diff
@@ -36,11 +41,32 @@ export class CalendarViewComponent implements OnInit {
...
+    if (month <= this.month.id) {
+      day = new Date(this.year + '-' + month + '-01').getDay();
+    } else {
+      day = new Date(this.year + '-' + month + '-01').getDay() + 1;
+    }
 
     return day;
   }
 
+  getPreviousMonth() {
```
Comment (tan-cy) [inline src/app/Components/calendar-view/calendar-view.component.ts]: this and line 59 arent get methods, rather they set the current month. something like `setToPreviousMonth` and `setToNextMonth` would be more appropriate

File: src/app/Components/calendar-view/calendar-view.component.ts [hunk 5]
```diff
@@ -36,11 +41,32 @@ export class CalendarViewComponent implements OnInit {
   }
 
   getWeekdayForFirstOfMonth(month: number) {
-    const day = new Date(this.year + '-' + month + '-01').getDay();
```
Comment (tan-cy) [inline src/app/Components/calendar-view/calendar-view.component.ts:41]: use `getUTCDay()` instead

**Review by tan-cy — APPROVED** [review]

**Pull request merged**